Source/WebKit/UIProcess/ios/WKTouchEventsGestureRecognizer.mm

```diff
@@ -284,10 +285,14 @@ - (void)_recordTouches:(NSSet<UITouch *> *)touches type:(WebKit::WKTouchEventTyp
     _lastTouchEvent.timestamp = approximateWallTime(touches.anyObject.timestamp);
 
     _lastTouchEvent.coalescedEvents = { };
+    _lastTouchEvent.predictedEvents = { };
 
     if (type == WebKit::WKTouchEventType::Change) {
         for (UITouch *coalescedTouch in coalescedTouches)
             _lastTouchEvent.coalescedEvents.append([self _coalescedTouchEventForTouch:coalescedTouch]);
+
+        for (UITouch *predictedTouch in predictedTouches)
+            _lastTouchEvent.predictedEvents.append([self _coalescedTouchEventForTouch:predictedTouch]);
```

Comment:
`_coalescedTouchEventForTouch:` might be a bit misleading for a reader... any other names? Maybe `touchEventFor(UI)Touch:`?